docs/docs/deployments.md

```diff
@@ -5,118 +5,110 @@ title: Application Deployments
 # Application Deployments
 
 Every change in the definition of applications or groups in Marathon is performed as a deployment.
-A deployment is a set of actions, that can do:
+A deployment is a set of actions, that can:
 
 - Start/Stop one or more applications
 - Upgrade one or more applications
 - Scale one or more applications
 
-Deployments are not instantly available - they take time. 
-A deployment is active in Marathon until the deployment is finished successfully.
+Deployments take time and are not instantly available. A deployment is considered active in Marathon until it has finished successfully.
```

Comment:
Doesn't this give the impression that a deployment is still available, but considered inactive, after it has finished successfully?

Marathon actually removes a deployment once it completes successfully.